src/Simplex/Chat/Archive.hs

```diff
@@ -94,53 +94,60 @@ copyDirectoryFiles fromDir toDir = do
 
 deleteStorage :: ChatMonad m => m ()
 deleteStorage = do
-  StorageFiles {chatDb, agentDb, filesPath} <- storageFiles
-  removeFile chatDb
-  removeFile agentDb
+  fs@StorageFiles {filesPath} <- storageFiles
```

Comment:
we could catch exception in deleteStorage, currently it crashes the app if file doesn't exist